src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -124,15 +126,11 @@ def extract_framework_task_model(model_id: str) -> Tuple[str, str, str]:
 
     Args:
         model_id (str): The model ID for which to extract the framework/task/model.
-
-    Raises:
-        ValueError: If the model ID cannot be parsed into at least 3 components seperated by
-            "-" character.
     """
     _id_parts = model_id.split("-")
 
     if len(_id_parts) < 3:
-        raise ValueError(f"incorrect model ID: {model_id}.")
```

Comment:
sanity-check: is this OK to change this behavior?

Reply:
We would need this change to allow filtering proprietary models that are not following the open-weights naming convention, so we don't want to raise here but allow ignoring framework/task if it's proprietary model. For existing open-weights models, this shouldn't affect CX, as this is raised only if our metadata is malformed.